### ISSUE TITLE
Actually Confirmed. Personal loot is the only loot method in BFA

### ISSUE THREAD

**Issue by SafeteeWoW** [evil-morfar/RCLootCouncil2#169]
TLDR, 
+ personal loot is the ONLY loot method in BFA. There is no way to change to another loot method in BFA. No more guild master loot.
+ The eligibility to trade the item is simple: If you have already owned a soulbound item on the same slot with equal or higher item level, the item looted by you will be tradable.

---

http://www.wowhead.com/news=283905/liveblog-of-the-april-26-battle-for-azeroth-beta-q-a-with-ion-hazzikostas

> Highlights

Here are some highlights of the Q&A:
+ Charge, Disengage and Infernal Strike will be removed from the GCD in a future build.
+ Azerite Power is now called Artifact Power
+ Conquest Gear will still have random rewards, but also a structure that will award pieces at set timetables.
+ **Personal loot will be used for all of BFA.**

> Any updates regarding loot methods in BFA?

Personal loot has really been the default choice in Legion. The only place where it's not is Guild Groups in raiding content. The choice was not meant to curb split raiding. The advantages are it has is more consistency over the entire game and it adds something personal. Group and Master Loot makes you the mercy at others. When you kill a boss, the reward is disconnected and you deserve the loot. Trading will allow for the social structures to do some things with loot.

> How will the item level and trading restriction for Personal Loot work mechanically going forward?

The highest item level piece in that slot that you have looted and soulbound will be recorded. You can trade anything up to and including that item level.

---

https://www.mmo-champion.com/content/7554-Developer-Q-A-with-Ion-Hazzikostas-Battle-for-Azeroth
> Loot Methods

+ Personal Loot has been the default in WoW for some time now. All of dungeons, PvP, the outdoor world are all personal loot.
+ The only place other loot methods are used are in guild raids.
+ This change wasn't made to stop high end split raids. That's something the team wants to do, but that wasn't the point of this change.
+ There is more personal agency and control as an individual of your fate.
+ There are some guilds that have awarded loot well for many years.
+ Group and Master Loot puts players at the mercy of others, often in ways that aren't fair.
+ Trial players in a new guild may not get anything for weeks because that is the guild policy.
+ When you kill a boss, it isn't about if there is something on the corpse that you want, but if someone is going to give you that loot. It disconnects the reward from killing the boss.
+ You deserve the loot for killing the boss.
+ It's not called Personal Loot anymore, it's just Loot. This is how loot works in the entire game.
+ Trading restrictions for loot will work the same as they do now for the most part. If it is lower than the highest item level piece you have ever looted and had soulbound for that slot, you can trade it.
